kafka-rest/src/main/java/io/confluent/kafkarest/response/CrnFactoryImpl.java

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
It seems like we should move this: https://github.com/confluentinc/ce-kafka/tree/master/ce-resource-names/src to an independent repo (probably https://github.com/confluentinc/crn) so that it can be used here.

Reply:
Acknowledged.